src/app/containers/ChartbeatAnalytics/canonical/index.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
 import { mount } from 'enzyme';
+import renderWithHelmet from '@bbc/psammead-test-helpers/dist/renderWithHelmet';
```

Comment:
Were using this direct from dist, which isnt a documented entry point and thus can be renamed/deleted without it being a breaking change. Can this be avoided?

Reply:
Looks like we can't use it directly, because `renderWithHelmet` isn't exported by `@bbc-psammead-test-helpers`, but we can use `shouldMatchSnapshot` instead.

It might be worth it to export `renderWithHelmet` in a future release of `@bbc-psammead-test-helpers`

Reply:
Fixed in new PR https://github.com/bbc/simorgh/pull/4269